ui/app/styles/components/calendar-widget.scss

```diff
@@ -0,0 +1,113 @@
+$dark-gray: #535f73;
+
+.calendar-content {
+  display: flex;
+  width: 500px;
+
+  > .menu {
+    margin-bottom: 0px;
+  }
+}
+.calendar-title {
+  color: $ui-gray-300;
+  text-transform: uppercase;
+  font-size: $size-7;
+  font-weight: $font-weight-semibold;
+
+  &.popup-menu-item {
+    padding: $size-10 $size-8;
+  }
+}
+.calendar-widget-dropdown {
+  @extend .button;
+  @extend .is-ghost;
+  @extend .has-icon-right;
+  border: 0;
+  color: $black;
+  transition: background-color $speed;
+  width: 150px;
+
+  &:hover {
+    background-color: $ui-gray-100;
+    border: 0;
+    color: $blue;
+  }
+
+  &.is-active {
+    background-color: $ui-gray-100;
+    border: 0;
+    color: $blue;
+  }
+}
+
+.select-year {
+  grid-area: select-year;
+  margin-left: 24px;
+  margin-top: 10px;
+  font-weight: $font-weight-bold;
+  display: flex;
+  justify-content: space-between;
+  align-items: first baseline;
+
+  > .padding-right {
+    padding-right: 26px;
+  }
+}
+
+.calendar-widget {
+  grid-area: calendar-widget;
+
+  > button {
+    &.is-month-list {
+      background-color: white;
+      color: black;
+      text-align: center;
+      border: 1px solid $ui-gray-200;
+      border-radius: 2px;
+    }
+    &.is-current-month {
+      border: 1px solid $ui-gray-900;
+    }
+    &:hover {
+      background-color: lighten($dark-gray, 30%);
+      color: white;
+      text-align: center;
+      cursor: pointer;
+    }
+    &.is-readOnly {
+      background-color: $ui-gray-100;
+      color: lighten($dark-gray, 30%);
+      pointer-events: none;
+    }
+    &.is-selected {
+      background-color: $blue-500;
```

Comment:
this never shows, even though we do add this class. Thoughts on if I should remove it or it keep it in case there is a lag?

Reply:
I think there's no harm in keeping it, especially if there is a lag

Reply:
thanks!